pom.xml

```diff
@@ -83,15 +83,15 @@
         <revision>2.2.2-SNAPSHOT</revision>
 
         <!-- Dependency Versions -->
-        <spring-cloud-commons.version>2.2.2.RELEASE</spring-cloud-commons.version>
-        <spring-cloud-netflix.version>2.2.2.RELEASE</spring-cloud-netflix.version>
-        <spring-cloud-openfeign.version>2.2.2.RELEASE</spring-cloud-openfeign.version>
-        <spring-cloud-bus.version>2.2.1.RELEASE</spring-cloud-bus.version>
-        <spring-cloud-gateway.version>2.2.2.RELEASE</spring-cloud-gateway.version>
+        <spring-cloud-commons.version>2.2.5.RELEASE</spring-cloud-commons.version>
+        <spring-cloud-netflix.version>2.2.5.RELEASE</spring-cloud-netflix.version>
+        <spring-cloud-openfeign.version>2.2.5.RELEASE</spring-cloud-openfeign.version>
+        <spring-cloud-bus.version>2.2.3.RELEASE</spring-cloud-bus.version>
+        <spring-cloud-gateway.version>2.2.5.RELEASE</spring-cloud-gateway.version>
         <spring-cloud-stream.version>Horsham.SR3</spring-cloud-stream.version>
```

Comment:
[spring-cloud-stream version should be SR8](https://github.com/spring-cloud/spring-cloud-release/blob/jenkins-releaser-config/hoxton_sr8.properties#L5)  @yuhuangbin